app/build.gradle

```diff
@@ -132,4 +132,6 @@ dependencies {
     exclude group: 'org.apache.httpcomponents', module: 'httpclient'
   }
   testCompile "org.jetbrains.kotlin:kotlin-reflect:$kotlin_version"
+
+  releaseCompile 'com.mapzen.android:leyndo:0.0.1-SNAPSHOT'
```

Comment:
Gradle tries to resolve dependencies marked as `releaseCompile` when you run `assembleDebug`. 

```
> Could not resolve all dependencies for configuration ':app:_releaseCompile'.
   > Could not find com.mapzen.android:leyndo:0.0.1-SNAPSHOT.
```

Wow.